client/app/scripts/liveblog-edit/freetype.service.js

```diff
@@ -166,7 +167,7 @@ function(angular) {
                     });
                     if (name) {
                         path2obj(scope[SCOPE_FREETYPEDATA], name);
-                        return '<input ng-model=' + makeAngularAttr(name, attr) + '/>';
+                        return '<freetype-text text=' + makeAngularAttr(name, attr) + ' validation="validation"></freetype-text>';
```

Comment:
Injecting a directive from a string variable defined in a service? Doesn't sound right to me